hotels/uploader.py

```diff
@@ -53,9 +53,10 @@ def write_list_to_hotel_model():
     """
     listed_data = get_csv_as_list(settings.HOTEL_CSV)
     for elem in listed_data:
-        city = City.objects.get(abbrv=elem[0])
         hotel, _ = Hotel.objects.get_or_create(
-            city=city,
             loc=elem[1],
             name=elem[2],
         )
+        if _:
+            hotel.objects.create(city=elem[0])

```

Comment:
Should be more like:

```suggestion
            hotel.city = City.objects.filter(abbrv=elem[0]).first()
```

Also, I used `filter`+`first` so that if the City does not exists, the city is set to `None` for the hotel.